iocage_lib/ioc_json.py

```diff
@@ -1931,6 +2010,17 @@ def json_set_value(self, prop, _import=False, default=False):
                 else:
                     key = key.replace("_", ".")
 
+                if key == 'cpuset':
+                    iocage_lib.ioc_common.logit(
+                        {
+                            'level': 'INFO',
+                            'message': 'cpuset changes to be implemented '
```

Comment:
Let's drop the `to be implemented` part of this string. So it just reads `cpuset changes require a jail restart`